qa/integration-tests/src/test/java/io/zeebe/broker/it/startup/BrokerReprocessingTest.java

```diff
@@ -27,6 +27,8 @@
 import io.zeebe.client.api.response.ActivateJobsResponse;
```

Comment:
I'm not sure what the changes here have to do with the PR - what am I missing?

Reply:
Tests were failing because after restart the requests are sent before the broker is in the topology.